libraries/AP_GPS/AP_GPS.cpp

```diff
@@ -580,3 +580,23 @@ AP_GPS::send_mavlink_gps2_rtk(mavlink_channel_t chan)
         drivers[1]->send_mavlink_gps_rtk(chan);
     }
 }
+
+// is this GPS instance configured
+bool AP_GPS::is_configured(uint8_t instance) const
+{
+    if (drivers[instance] != NULL && state[instance].status != NO_GPS) {
```

Comment:
Wouldnt GPS status be none?
On Jan 24, 2016 11:10 AM, "WickedShell" <notifications@github.com> wrote:

> In libraries/AP_GPS/AP_GPS.cpp
> <https://github.com/diydrones/ardupilot/pull/3546#discussion_r50640310>:
>
> > @@ -580,3 +580,23 @@ AP_GPS::send_mavlink_gps2_rtk(mavlink_channel_t chan)
> >          drivers[1]->send_mavlink_gps_rtk(chan);
> >      }
> >  }
> > +
> > +// is this GPS instance configured
> > +bool AP_GPS::is_configured(uint8_t instance) const
> > +{
> > +    if (drivers[instance] != NULL && state[instance].status != NO_GPS) {
>
> This will always return false on instance 2 of single GPS operation
> (assuming the GPS is plugged into the normal GPS port on a Pixhawk) (I
> believe)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/diydrones/ardupilot/pull/3546/files#r50640310>.
>